test/content-legacy/deprecation-warning.html

```diff
@@ -17,7 +17,7 @@
     <test-fixture id="juicy-html-with-content">
         <template>
             <!-- nest to workaround test-fixture bug -->
-            <div><template is="juicy-html" content="/mock/smth">
+            <div><template is="juicy-html" content="../mock/smth.html">
```

Comment:
Here you use relative path `../`, but in `test/content-legacy/no-content.html` you use an absolute path `/`. This should be consistent.

Reply:
solved